src/paulscode/android/mupen64plusae/GameActivity.java

```diff
@@ -61,6 +61,7 @@ public void onWindowFocusChanged( boolean hasFocus )
     @Override
     protected void onCreate( Bundle savedInstanceState )
     {
+        super.setTheme( android.support.v7.appcompat.R.style.Theme_AppCompat_NoActionBar );
```

Comment:
The ActionBar is no longer used, so this requires an update to the theme. Once all of the activities are switched over to the NoActionBar theme we can set it in the manifest file and remove these lines.